File: Competitive Coding/C/Keshav Jain/Bubble Sort.c
#include<stdio.h>
int main()
{
    int i, j, temp, n, a[50];
    printf("Enter no. of Elements: ");
    scanf("%d",&n);
    printf("Enter Array\n");
    for(i=0 ; i<n ; i++)
        scanf("%d",&a[i]);
    for(i=0 ; i<n-1 ; i++)
    {
        for(j=0 ; j<n-i-1 ; j++)
        {
            if(a[j+1] < a[j])
            {
                temp = a[j+1];
                a[j+1] = a[j];
                a[j] = temp;
            }
        }
    }
    printf("Sorted Array:\n");
    for(i=0 ; i<n ; i++)
        printf("%d\n",a[i]);
    return 0;
}

File: Competitive Coding/Java/Anchal Goyal/bubbleSort.java
import java.util.Scanner;
// Other imports go here
// Do NOT change the class name
class bubbleSort
{
    static void bubbleSort(int arr[],int n)
    {
int temp;
        for(int i=0;i<n-1;i++)
        {
            for(int j=0;j<n-1;j++)
             {
                if(arr[j]>arr[j+1])
                {
                  temp=arr[j];
                  arr[j]=arr[j+1];
                  arr[j+1]=temp;
                }
             }
        }
    }
   
    public static void main(String[] args)
    {
        // Write your code here
        Scanner s=new Scanner(System.in);
        System.out.print("enter no of elements");
        int n=s.nextInt();
        int arr[]=new int[n];
        for(int i=0;i<n;i++)
        {
            arr[i]=s.nextInt();
        }
        bubbleSort(arr,n);
        for(int i=0;i<n;i++)
        {
            System.out.print(arr[i]+" ");
        }
    }
    }